profiles/common/modules/features/nexteuropa_core/nexteuropa_core.tokens.inc

```diff
@@ -27,13 +27,12 @@ function nexteuropa_core_tokens($type, $tokens, array $data = array(), array $op
     $node = $data['node'];
     foreach ($tokens as $name => $original) {
       if (strstr($name, ':parents-uri') !== FALSE) {
-        list($field_name, $delta,) = explode(':', $name);
+        list($field_name) = explode(':', $name);
         $field_name = str_replace('-', '_', $field_name);
         if (isset($node->{$field_name})) {
           $node_wrapper = entity_metadata_wrapper('node', $node);
-          $terms = $node_wrapper->{$field_name}->value();
-          if (isset($terms[$delta - 1])) {
-            $term = $terms[$delta - 1];
+          $term = $node_wrapper->{$field_name}->value();
+          if (!empty($term)) {
             $parents = array();
             foreach (entity_metadata_taxonomy_term_get_properties($term, array(), 'parents_all') as $tid) {
```

Comment:
Instead of "entity_metadata_taxonomy_term_get_properties", we should use taxonomy_get_parents_all. it retrieve all parent taxonomy terms what entity_metadata_taxonomy_term_get_properties and that could avoid to do another taxonomy_term_load in the loop.